src/suites/cts/gpu_test.ts

```diff
@@ -14,6 +14,24 @@ export class GPUTest extends Fixture {
     this.queue = this.device.getQueue();
   }
 
+  async finalize(): Promise<void> {
```

Comment:
Why is this in GPUTest? It seems to go with the init of ValidationTest so it could go in a finalize there no?

Reply:
See https://github.com/gpuweb/cts/pull/25#discussion_r322960625